e2e-tests/IIoTPlatform-E2E-Tests/TestExtensions/IIoTPlatformTestContext.cs

```diff
@@ -157,5 +162,20 @@ public void Dispose()
 
         string ITestEventProcessorConfig.TestEventProcessorPassword => GetStringOrDefault(TestConstants.EnvironmentVariablesNames.TESTEVENTPROCESSOR_PASSWORD,
             () => throw new Exception("Test Event Processor Password is not provided."));
+
+        string IContainerRegistryConfig.ContainerRegistryServer => GetStringOrDefault(TestConstants.EnvironmentVariablesNames.PCS_CONTAINER_REGISTRY_SERVER,
+            () => string.Empty);
+
+        string IContainerRegistryConfig.ContainerRegistryUser => GetStringOrDefault(TestConstants.EnvironmentVariablesNames.PCS_CONTAINER_REGISTRY_USER,
+            () => string.Empty);
+
+        string IContainerRegistryConfig.ContainerRegistryPassword => GetStringOrDefault(TestConstants.EnvironmentVariablesNames.PCS_CONTAINER_REGISTRY_PASSWORD,
+            () => string.Empty);
+
+        string IContainerRegistryConfig.ImagesNamespace => GetStringOrDefault(TestConstants.EnvironmentVariablesNames.PCS_IMAGES_NAMESPACE,
+            () => string.Empty);
+
+        string IContainerRegistryConfig.ImagesTag => GetStringOrDefault(TestConstants.EnvironmentVariablesNames.PCS_IMAGES_TAG,
+            () => throw new Exception("Images tag not provided"));
```

Comment:
This will still cause test runs to fail as `PCS_IMAGES_TAG` is not provided from the pipeline.

Reply:
@karok2m agreed, updating the pipeline to successful use the functionality should be part of next task, with the e2e test for standalone